devlab/tests/config.py

```diff
@@ -458,6 +485,19 @@
      'server_group': 'admin_server_group'}
 ]
 """VM's to create/delete"""
+
+dst_vms = [
+    {'name': 'tn5server1', 'image': 'image1', 'flavor': 'flavorname2',
+     'tenant': 'tenant5', 'nics': [{'net-id': 'tenant5net',
+                                    'v4-fixed-ip': '122.2.2.100'}]},
+    {'name': 'tn5server2', 'image': 'image1', 'flavor': 'flavorname2',
+     'tenant': 'tenant5', 'nics': [{'net-id': 'tenant5net2',
+                                    'v4-fixed-ip': '123.2.2.102'}]},
+    {'name': 'tn5server3', 'image': 'image1', 'flavor': 'flavorname2',
+     'nics': [{'net-id': 'tenant5net', 'v4-fixed-ip': '122.2.2.101'}],
+     'tenant': 'tenant5'},
+]
+
```

Comment:
Please add documentation describing test cases added here. Reach out to @plomakin for help